src/corner_kick/yarn.lock

```diff
@@ -0,0 +1,6698 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Should this file be included? Seems like it's autogenerated during build or something.

Reply:
Yes, this file should be included: https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/